includes/definitions/chatsworth.yaml

```diff
@@ -0,0 +1,8 @@
+os: chatsworth
+text: 'Chatsworth PDU'
+type: power
+icon: chatsworth
+nobulk: 1
+discovery:
+    - sysObjectId:
+        - .0.10.43.6.1.4.1
```

Comment:
Is that genuinely the sysObjectID, if so it's probably worth raising to them as that's just, erm, whacky.